api/client/info.go

```diff
@@ -35,7 +35,9 @@ func (cli *DockerCli) CmdInfo(args ...string) error {
 
 	fmt.Fprintf(cli.out, "Containers: %d\n", info.Containers)
 	fmt.Fprintf(cli.out, "Images: %d\n", info.Images)
-	fmt.Fprintf(cli.out, "Engine Version: %s\n", info.ServerVersion)
+	if info.ServerVersion != "" {
+		fmt.Fprintf(cli.out, "Engine Version: %s\n", info.ServerVersion)
+	}
```

Comment:
Should this use `ioutils.FprintfIfNotEmpty`? (I just spotted that the line below)